src/migrations/20190705144236-create-user-table.ts

```diff
@@ -0,0 +1,32 @@
+import { DataTypes, QueryInterface } from "sequelize";
+
+export default {
+  up: (queryInterface: QueryInterface): Promise<void> => {
+    return queryInterface.createTable("User", {
+      createdAt: {
+        allowNull: false,
+        type: new DataTypes.DATE()
+      },
+      deletedAt: new DataTypes.DATE(),
+      fc: {
```

Comment:
use a name for this field that is easier to understand + add a comment explaining each field

Reply:
```suggestion
      fiscalCode: {
```